labs/coding-101-rest-basics-ga/5.md

```diff
@@ -30,7 +30,7 @@ On OS X, to use Python 3.4, you may need to accept the SSL certificate before ca
 
 2. Set up your request to retrieve a token. In the Postman fields, enter the following information:
 
-   ![](/posts/files/coding-101-rest-basics-ga/assets/images/postman1.png)
+   ![](assets/images/postman1.png)
```

Comment:
Oh, I bet the screenshots have to be retaken. Do you have a copy of Postman installed? I can send over the collection you can use to do the screenshots.